tests/generic_config_updater/test_portchannel_interface.py

```diff
@@ -0,0 +1,348 @@
+import logging
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.generic_config_updater.gu_utils import apply_patch, expect_op_success, expect_op_failure
+from tests.generic_config_updater.gu_utils import generate_tmpfile, delete_tmpfile
+from tests.generic_config_updater.gu_utils import create_checkpoint, delete_checkpoint, rollback_or_reload
+from tests.generic_config_updater.gu_utils import create_path, check_show_ip_intf
+
+# Test on t0 topo to verify functionality and to choose predefined variable
+# "PORTCHANNEL_INTERFACE": {
+#     "PortChannel0001": {},
+#     "PortChannel0001|10.0.0.56/31": {},
+#     "PortChannel0001|FC00::71/126": {},
+#     "PortChannel0002": {},
+#     "PortChannel0002|10.0.0.58/31": {},
+#     "PortChannel0002|FC00::75/126": {},
+#     "PortChannel0003": {},
+#     "PortChannel0003|10.0.0.60/31": {},
+#     "PortChannel0003|FC00::79/126": {},
+#     "PortChannel0004": {},
+#     "PortChannel0004|10.0.0.62/31": {},
+#     "PortChannel0004|FC00::7D/126": {}
+# }
+
+pytestmark = [
+    pytest.mark.topology('t0'),
```

Comment:
>t0 [](http://example.com/codeflow?start=26&length=2)

also test on t1-lag?

Reply:
Can you find me a t1-lag topo? I can check if the test can be adopted.

Reply:
Maybe add t1-lag in another PR. 
In this testcase, we are actually modifying the interface, like ip and attributes(mtu, min-link). Topo should not make any difference right?